.php_cs

```diff
@@ -1,9 +1,9 @@
 <?php
 
 /*
- * This file is part of Gush.
+ * This file is part of Gush package.
  *
- * (c) Luis Cordova <cordoval@gmail.com>
+ * (c) 2013-2015 Luis Cordova <cordoval@gmail.com>
```

Comment:
Wrong header, we don't include the year ranges anymore since causes to many changes.